lib/compat/wordpress-6.0/block-template-utils.php

```diff
@@ -104,11 +104,11 @@ function gutenberg_generate_block_templates_export_file() {
 	// If a version is defined, add a schema.
 	if ( $theme_json_raw['version'] ) {
 		global $wp_version;
-		$theme_json_version = substr( $wp_version, 0, strpos( $wp_version, '-' ) );
+		$theme_json_version = 'wp/' . substr( $wp_version, 0, 3 );
```

Comment:
What happens when WordPress reaches version 10?

@scruffian: I spotted this today in Core while looking at other things. Can we fix this? Something like this is enough:

```diff
-		$theme_json_version = 'wp/' . substr( $wp_version, 0, 3 );
+		$theme_json_version = 'wp/' . substr( $version, 0, strpos( $version, '-' ) );
```

Reply:
Created a core patch here: https://core.trac.wordpress.org/ticket/60486